kickstart/provision_atomic.erb

```diff
@@ -0,0 +1,40 @@
+<%#
+kind: provision
+name: Atomic Kickstart
+oses:
+- CentOS-Atomic 7
+- Fedora-Atomic 22
+- RedHat-Atomic 7
+%>
+lang <%= @host.params['lang'] || 'en_US.UTF-8' %>
+keyboard <%= @host.params['keyboard'] || 'us' %>
+timezone --utc <%= @host.params['time-zone'] || 'UTC' %>
+
+# Partition table should create /boot and a volume atomicos
+<% if @dynamic -%>
+%include /tmp/diskpart.cfg
+<% else -%>
+<%= @host.diskLayout %>
+<% end -%>
+
+
+bootloader --timeout=3
+<% if @host.os.name.match /.*fedora.*/i -%>
+ostreesetup --nogpg --osname=fedora-atomic --remote=fedora-atomic --url=<%= @host.operatingsystem.medium_uri(@host) %>/content/repo/ --ref=fedora-atomic/f<%= @host.os.major %>/<%= @host.architecture %>/docker-host
+<% else -%>
+ostreesetup --nogpg --osname=rhel-atomic-host --remote=rhel-atomic-host --url=file:///install/ostree --ref=rhel-atomic-host/<%= @host.os.major %>/<%= @host.architecture %>/standard
```

Comment:
Not 100% sure, but I don't think the contents of `osname` and `remote` options has any effect on anything in this case.